src/main/java/org/mockito/Mockito.java

```diff
@@ -2228,6 +2230,22 @@ public static VerificationMode times(int wantedNumberOfInvocations) {
     }
 
     /**
+     * Allows verifying exact number of invocations. E.g:
+     * <pre class="code"><code class="java">
+     *   AtomicInteger counter = new AtomicInteger();
+     *   counter.set(2);
+     *   verify(mock, times(counter)).someMethod("some arg");
+     * </code></pre>
```

Comment:
I don't see any benefit here. It clutters the API without bringing value, why not just 

```
verify(mock, times(counter.get())).someMethod("some arg");
```